### ISSUE TITLE
Format fails on certain operator <= >= 

### ISSUE THREAD

**Issue by cdekok** [Benestar/asparagus#20]
It will add a space in the operator so <= will become < =

**Comment by Benestar** [comment]
meh, I see. Need to improve the regexes... Anyways, the formatter is quite a mess and should perhaps be replaced by something already implemented ;-)

**Comment by cdekok** [comment]
Yes I think the regex method will break on many things like values in a select.
It would probably need some sort of tokenizer to do formatting more reliable.
